package.json

```diff
@@ -91,6 +91,7 @@
     "src/hooks.ts",
     "src/htmldomapi.ts",
     "src/is.ts",
+    "src/jsx.ts",
```

Comment:
Is this a fix? If so, an issue should be opened and this fixed in a separate PR, if you ask me. Same for the rest of the changes in this file.